### DISCUSSION TITLE
Mute 상태 에서도 마이크 조절 가능

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#48] [제안하기]
Mute에서는 마이크 조절이 안되는게 spec이었다니!! 조절이 불가능 한 줄 알았어요.

Mute 상태에서도 마이크 음량 조절이 되었으면 좋겠는데, 안된다면 마우스 음량 조절 클릭했을때 mute를 풀어야 변경 가능하다고 툴팁이 뜨면 좋을 것 같습니다

**Comment by ACT-TJ** [comment]
안녕하세요.

지금해보니 또 되는데.. 뭔가 그때 잠시 Mute 아닐때도 안되는 상황에서 문제였었나봐요. 
현재는 Mute일때도 게이지 조절이 되는 것을 확인했어요. 혼선을 드려 죄송합니다.

또 관련 문제 있다면 글 남겨주세요~~!
